src/components/MultiSelect/MultiSelect.js

```diff
@@ -103,6 +110,10 @@ export default class MultiSelect extends React.Component {
     type: 'default',
     light: false,
     title: false,
+    listBoxMenuIconTranslationIds: {
```

Comment:
Wondering if we can make it a callback style as some other components do.

Reply:
makes sense, refactored to take a callback function rather than a config object